ceph.spec.in

```diff
@@ -14,7 +14,7 @@
 #
 # Please submit bugfixes or comments via http://tracker.ceph.com/
 # 
-%bcond_with ocf
+%bcond_without ocf
```

Comment:
@tchaikov Yes, and for example why the `ceph-osd` package is called that and not merely `osd` :-P